### PR TITLE
Add carousel component to homepage

### LINKED ISSUE #72: add carousel component
add reusable carousel component

### PR DESCRIPTION
**Pull request by vicalden23**
closes #72

### CODE REVIEW

File: client/src/containers/Homepage.js [hunk 1]
```diff
@@ -1,50 +1,68 @@
...
+import HorizontalGrid from '../components/HorizontalGrid'
 
 import './Homepage.css';
 
 
 class Homepage extends Component {
   componentDidMount() {
-    console.log(this.props.fetchTodaysMeals());
+
+    console.log(this.props.fetchUpcomingMeals());
```
Comment (gzeegz) [inline client/src/containers/Homepage.js]: Let's remove the `console.log` and just call `this.props.fetchUpcomingMeals`.

File: client/src/containers/Homepage.js [hunk 2]
```diff
@@ -1,50 +1,68 @@
...
+
   }
 
   render() {
+    let dates = Object.keys(this.props.upcomingMeals).sort(function(a, b) {
+      return new Date(a).getTime() - new Date(b).getTime();
+    });
+    console.log(this.props.upcomingMeals);
+    console.log(dates);
```
Comment (gzeegz) [inline client/src/containers/Homepage.js]: The `console.log`s should be removed.

**Review by gzeegz — CHANGES_REQUESTED** [review]
Small changes need to be made. Need to remove `console.log`s.

File: client/src/actions/mealsActions.js [hunk 3]
```diff
@@ -9,6 +9,7 @@ export const FETCH_MEALDETAIL = 'FETCH_MEALDETAIL';
 export const FETCH_MEALS_BY_CHEF = 'FETCH_MEALS_BY_CHEF';
 export const GET_NEAR_BY_MEALS = 'GET_NEAR_BY_MEALS';
 export const FETCH_TODAYS_MEALS = 'FETCH_TODAYS_MEALS';
+export const FETCH_UPCOMING_MEALS = 'FETCH_UPCOMING_MEALS';
```
Comment (obay-mardini) [inline client/src/actions/mealsActions.js]: I would delete fetch today and tomorrow's meals logic (reducers and actions creators) bc we are not going to use them anymore

**Review by gzeegz — APPROVED** [review]
LGTM

File: client/src/containers/Homepage.js [hunk 4]
```diff
@@ -1,50 +1,68 @@
 import React, { Component } from 'react';
-import { connect } from 'react-redux';
-import { fetchTodaysMeals } from '../actions/index';
+import _ from 'lodash';
+
+import Carousel from 'nuka-carousel';
+import Moment from 'react-moment';
```
Comment (obay-mardini) [inline client/src/containers/Homepage.js]: Please could you get rid of Moment import statement bc we are not using it in this file, and all Moment related code i.e package.json and other places

**Review by obay-mardini — CHANGES_REQUESTED** [review]

File: client/src/actions/mealsActions.js (hunk 3 above)
Comment (vicalden23) [reply client/src/actions/mealsActions.js]: 👍

File: client/src/containers/Homepage.js (hunk 4 above)
Comment (vicalden23) [reply client/src/containers/Homepage.js]: 👍

**Review by obay-mardini — APPROVED** [review]

**Review by obay-mardini — CHANGES_REQUESTED** [review]
LGTM, but could you resolve the merge conflict

**Comment by obay-mardini** [comment]
closes #127

**Review by obay-mardini — APPROVED** [review]

**Pull request merged**